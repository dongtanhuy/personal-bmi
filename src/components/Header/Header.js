import React from 'react';
import './Header.scss';

const Header = () => (
  <h1 className="header">Personal Body Mass Index</h1>
);

export default Header;
