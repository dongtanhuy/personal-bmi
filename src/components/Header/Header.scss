@import 'styles/main.scss';
.header {
  line-height: 110%;
  font-size: 4rem;
}