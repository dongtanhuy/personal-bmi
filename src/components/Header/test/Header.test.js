import React from 'react';
import { render } from '@testing-library/react';
import Header from '../Header';

test('renders App', () => {
  const { container } = render(<Header />);
  expect(container).toMatchSnapshot();
});

