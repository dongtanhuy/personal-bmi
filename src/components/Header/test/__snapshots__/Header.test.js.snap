// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App 1`] = `
<div>
  <h1
    class="header"
  >
    Personal Body Mass Index
  </h1>
</div>
`;
